main.js

```diff
@@ -0,0 +1,155 @@
+let reset = document.getElementById('reset');
+
+let gameBoard = document.getElementsByClassName('board')
+let row1 = document.getElementById('row1')
+let cell1 = document.getElementById('cell1')
+let cell2 = document.getElementById('cell2')
+let cell3 = document.getElementById('cell3')
+
+let row2 = document.getElementById('row2')
+let cell4 = document.getElementById('cell4')
+let cell5 = document.getElementById('cell5')
+let cell6 = document.getElementById('cell6')
+
+let row3 = document.getElementById('row3')
+let cell7 = document.getElementById('cell7')
+let cell8 = document.getElementById('cell8')
+let cell9 = document.getElementById('cell9')
+
+let aiPlayer = "O";
+let player = "X";
+let Winner = null;
+
+let board = [
+    cell1, cell2, cell3,
+    cell4, cell5, cell6,
+    cell7, cell8, cell9
+]
+
+let prefferedMove = {
+    move1: cell1,
+    move2: cell2,
+    move3: cell3,
+    move4: cell4,
+    move5: cell5,
+    move6: cell6,
+    move7: cell7,
+    move8: cell8,
+    move9: cell9,
+}
+
+//-----------------------------------------------------------------------------------------------------------------------------
+
+function startTimer() {
+    let seconds = 0;
+    timer = setInterval(function () {
+        seconds++;
+        document.getElementById("seconds").innerText = seconds % 60;
+        document.getElementById("minutes").innerText = parseInt(seconds / 60);
+        document.getElementById("hours").innerText = parseInt(seconds / 3600);
+    }, 1000);
+}
+
+function stopTimer() {
+    clearInterval(timer);
+}
+
+//-----------------------------------------------------------------------------------------------------------------------------
+// Function for ai game play \\
+
+function available() {
+    for (let cell of board) {
+        if (cell.textContent = "") {
+        }
+    }
+}
+
+player1.addEventListener('click', () => {
+    event.target.disabled = true;
+    status.textContent = 'Player X, your turn!'
+    if (status.textContent = 'Player O, your turn') {
+
+    }
+})
+
+//-----------------------------------------------------------------------------------------------------------------------------
+// Function to reset board \\
+
+reset.addEventListener('click', () => {
+    window.location.reload()
+})
+
+//-----------------------------------------------------------------------------------------------------------------------------
+// Functions for player vs player mode \\
+
+player2.addEventListener('click', () => {
+    startTimer()
+    event.target.disabled = true;
+    status.textContent = 'Player X, your turn!'
+})
+let counter = 0
+twoPlayer()
+function twoPlayer() {
+    for (let cell of board) {
+        cell.addEventListener('click', () => {
+            counter = counter + 1
+            if (counter === 9 && Winner === null) {
+                status.textContent = "Its a tie."
+                stopTimer()
+            }
+            if (cell.textContent === "X" || cell.textContent === "O") {
```

Comment:
Remember that with the **if** statement, once that hits a set of guidelines you've given it, it won't continue looking to do anything else within that function.  So when you say...
`if (cell.textContent === "X" || cell.textContent === "O" {
 status.textContent = "Invalid move! Pick an empty spot."
            }`
...you're telling the program that **if** either of these happen, print that and stop running. You're not going to hit anything **else**, because it thinks it's already completed it's task.